### PR TITLE
pin-2709: SocketTimeoutException during rate limiting

### PR DESCRIPTION
**Pull request by galales**
This is a tentative solution for a possible cause to the rate limiting timeout

### CODE REVIEW

**Comment by TonioGela** [comment]
What's the state of this PR @galales?

**Comment by galales** [comment]
> What's the state of this PR @galales?

Waiting for love in form of approval 🤣 
I tried the deploy to test the solution before merging, but I forgot we had a check to block it.

We can just proceed with the merge, it's something IMO we should do anyway

**Review by TonioGela — APPROVED** [review]

**Review by micdes-pagopa — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**
